src/helpers/reservations.ts

```diff
@@ -614,6 +619,11 @@ const bookReservation = async (person, reservation, increaseAmount) => {
       );
   }
 
+  if (decreaseAmount) {
+    additionalAmount =
+      -1 * (1 + Math.random() * reservation.amount.value * 0.98);
```

Comment:
ar fi bine sa fie predictibila suma in loc de random ca sa putem scrie teste e2e cu un assert cunoscut

Reply:
in ambele cazuri (si increase si decrease)

Reply:
ex: increase / decrease by 50% or something like that

Reply:
https://github.com/etvascom/mock-solaris/pull/30/commits/aec5b784f77d1927de34ae70f0eee1e62e92f917